### PR TITLE
SSR Pandora

### PR DESCRIPTION
**Pull request by appinteractive**
Make sure the following is working: 
- [x]  running `yarn build` and then running `env $WEB_APP_PORT=8081 yarn start` will start the app on port 8081 instead of 3000 (we use now dynamic .env files) (see #25)
- [x]  login, logout etc. are working like expected without any spontanious logouts
- [x]  user import in admin is working via csv

### CODE REVIEW

**Comment by appinteractive** [comment]
DO NOT MERGE!

File: on-build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# it writes the environment variables to the config files
+# this is needed as nuxtjs replaces all mentions of process.env
+# with static vars at build time but we need them fresh at deploy time
+
+# set the build datetime
+export BUILD_DATE="$(date +'%Y-%m-%d %T')"
+
+# create .env from .env.tmp and fill in build vars
+# while swill keep deploy time variables intact
+envsub --protect .env.tmp .env
```
Comment (chrisSCM) [inline on-build.sh]: ok got it. (please ignore my former comment on .env.tmp filename)

**Comment by appinteractive** [comment]
@visualjerk do you mean a cookie from the last version or a cookie older then 24h? Any suggestions or possible fix?

**Comment by appinteractive** [comment]
@visualjerk are you sure that the issue was not caused be a non existend (old) user in your test system?

**Pull request merged**